<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfSelect</class>
 <widget class="QDialog" name="ConfSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>168</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>526</width>
    <height>168</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>526</width>
    <height>168</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Выбрать конфигурацию</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tgca.qrc">
    <normaloff>:/pictogram/folder_explore_2684.png</normaloff>:/pictogram/folder_explore_2684.png</iconset>
  </property>
  <widget class="QComboBox" name="comboBoxConfigurations">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>STD_BC_NOINT_QPSK_CODEC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>STD_BC_INT_QPSK_CODEC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>STD_BC_NOINT_QPSK_NOCODEC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>STD_BC_INT_QPSK_NOCODEC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>STD_BC_NOINT_QAM16_CODEC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>STD_BC_NOINT_QAM16_NOCODEC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>STD_BC_NOINT_QAM64_CODEC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>STD_BC_NOINT_QAM64_NOCODEC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>STD_RT_NOINT_QPSK_CODEC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>STD_RT_NOINT_QPSK_NOCODEC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>STD_RT_INT_QAM16_CODEC_EN1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>STDU_BC_NOINT_QPSK_CODEC</string>
    </property>
   </item>
  </widget>
  <widget class="QRadioButton" name="radioButtonChoose">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Выбрать стандартную конфигурацию</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonChooseFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>231</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Загрузить конфигурацию из файла</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPathToConf">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBrowse">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConfirm">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>95</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать</string>
   </property>
   <property name="icon">
    <iconset resource="tgca.qrc">
     <normaloff>:/pictogram/package-installed-updated_5020.png</normaloff>:/pictogram/package-installed-updated_5020.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
   <property name="icon">
    <iconset resource="tgca.qrc">
     <normaloff>:/pictogram/close_5496.png</normaloff>:/pictogram/close_5496.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="tgca.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonConfirm</sender>
   <signal>pressed()</signal>
   <receiver>ConfSelect</receiver>
   <slot>onConfirm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonBrowse</sender>
   <signal>pressed()</signal>
   <receiver>ConfSelect</receiver>
   <slot>onBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>pressed()</signal>
   <receiver>ConfSelect</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonChoose</sender>
   <signal>toggled(bool)</signal>
   <receiver>ConfSelect</receiver>
   <slot>onRadioChoose(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>configurate(QString,int*)</signal>
  <slot>onBrowse()</slot>
  <slot>onConfirm()</slot>
  <slot>onRadioChoose(bool)</slot>
 </slots>
</ui>
